bin/cli.js

```diff
@@ -35,10 +35,16 @@ prettyCLI
         desc: 'Theme path points to a zip file'
     })
     .boolean('-1, --v1', {
-        desc: 'Check theme for Ghost 1.0 compatibility, instead of 2.0'
+        desc: 'Check theme for Ghost 1.0 compatibility'
+    })
+    .boolean('-2, --v2', {
+        desc: 'Check theme for Ghost 2.0 compatibility'
+    })
+    .boolean('-3, --v3', {
+        desc: 'Check theme for Ghost 3.0 compatibility'
     })
     .boolean('-c, --canary', {
-        desc: 'Check theme for upcoming version compatibility, instead of 2.0'
+        desc: 'Check theme for upcoming Ghost 3.0 compatibility, instead of 2.0'
     })
```

Comment:
to be more generic I would maybe go with:
"Check theme for upcoming Ghost version compatibility"

Reply:
Agree and prefer the generic wording :+1: `Ghost 1.0` etc. were used in other commands so didn't know what's the best way here :wink: